src/SFA.DAS.ApplyService.Application/Apply/Assessor/GetPageReviewOutcomeRequest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MediatR;
+using SFA.DAS.ApplyService.Domain.Apply;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Assessor
+{
+    public class GetPageReviewOutcomeRequest : IRequest<PageReviewOutcome>
+    {
+        public GetPageReviewOutcomeRequest()
+        {
+
+        }
+
+        public GetPageReviewOutcomeRequest(Guid applicationId,
+                                                    int sequenceNumber,
+                                                    int sectionNumber,
+                                                    string pageId,
+                                                    int assessorType,
+                                                    string userId)
+        {
+            ApplicationId = applicationId;
+            SequenceNumber = sequenceNumber;
+            SectionNumber = sectionNumber;
+            PageId = pageId;
+            AssessorType = assessorType;
+            UserId = userId;
+        }
+
+        public Guid ApplicationId { get; set; }
```

Comment:
Can the properties be made Get only if they're being set in the constructor?  Same applies to the morder Request classes.

Reply:
Setters removed => PR 549